src/logic/claims/data.ts

```diff
@@ -192,3 +210,53 @@ export function getCurrentDeadline(claim: Claim) {
   }
   return claim.deadline;
 }
+
+async function getClaimContractData(contract: ClaimsManagerWithKlerosArbitration, claimEvents: CreatedClaimEvent[]) {
+  const calls = claimEvents.map((event) => {
+    return contract.interface.encodeFunctionData('claims', [event.args.claimIndex]);
+  });
+
+  const encodedResults = await contract.callStatic.multicall(calls);
+
+  return encodedResults.map((res) => {
+    return contract.interface.decodeFunctionResult('claims', res);
+  });
+}
+
+async function getDisputeContractData(
```

Comment:
The way we load the dispute data will likely change once the MockKlerosArbitrator resembles the KlerosLiquid contract: https://github.com/api3dao/claims-manager/issues/25